### DISCUSSION TITLE
Select create option button more visible

### DISCUSSION THREAD

**Discussion by vincent-tarrit** [filamentphp/filament#5899] [Ideas]
Hello, Hope you are well ! 

I was thinking that the + icon to create a new record on select.createOption is not that visible. 

Do you think it would be possible to have a way to customize it as a normal action button ? Orange and huge haha 

Thank you ! and would be happy to PR it if you explain me a bit how I could achieve it!

Vincent

**Comment by danharrin** [comment]
Try `createOptionAction(fn ($action) => $action->button())`
